pkg/new-ui/v1beta1/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+  - kimwnasptd
+reviewers:
+  - andreyvelich
+  - kimwnasptd
```

Comment:
Please remove myself, since I am in the root OWNERS file and you can just add yourself in approvers list.
```suggestion
approvers:
  - kimwnasptd
```

Reply:
Sure!